io/types.d.ts

```diff
@@ -1,57 +1,203 @@
+import { Module } from '@nuxt/types';
 import * as SocketIOClient from 'socket.io-client';
+import Vue from 'vue';
+
+/**
+ * The format of each entry in mutations can be:
+ *
+ *   1. A single name string - the event name acts as the mutation
+ *   2. A string with a double-dashed arrow - the left side of the arrow is the event name, the right side is the mutation
+ *   3. An object - the object key is the event, the value is the mutation
+ */
+type MutationNotation = string | Record<string | string>;
```

Comment:
I think the Record should be: Record<string, string> ?

Reply:
Good catch, thanks! :detective: